setup.cfg

```diff
@@ -100,10 +100,14 @@ doctest_plus = enabled
 text_file_format = rst
 open_files_ignore = "astropy.log" "/etc/hosts" "*.ttf"
 remote_data_strict = true
-addopts = -p no:warnings --doctest-rst
+addopts = --doctest-rst
 asdf_schema_root = astropy/io/misc/asdf/data/schemas
 xfail_strict = true
 qt_no_exception_capture = 1
+filterwarnings =
+    ignore:unclosed file:ResourceWarning
+    ignore:unclosed <socket:ResourceWarning
+    ignore:numpy.ufunc size changed:RuntimeWarning
```

Comment:
Note to self: Need to add another line to ignore this one, as I don't know how to silence it otherwise:
```
astropy/utils/timer.py:177
  .../astropy/utils/timer.py:177: AstropyDeprecationWarning:
  The timefunc function is deprecated and may be removed in a future version.
          Use astroquery.utils.timer.timefunc instead.
    @timefunc(num_tries=1, verbose=False)
```

This is because I am decorating a method with deprecated decorator. But both the method's class and the decorator are deprecated and I purposely excluded them from API doc collection, etc (you can check it on `master`). Still this warning pops up. 🤷‍♀